features/support/setup_browser.rb

```diff
@@ -7,7 +7,7 @@
 require 'base64'
 require 'capybara'
 require 'capybara/cucumber'
-require File.join(File.dirname(__FILE__), 'cobbler_test')
+require_relative 'cobbler_test'
```

Comment:
+1

Reply:
LOL for the description of the style. But please defuckify your comments :-).

I am not sure about the require_relative not being needed. I suggest that if you want to experiment with this, you do it in a separate pull request.

Reply:
Create a fixme commenr above the code so we can adress it

Reply:
ok

Reply:
@Bischoff  where is the fixme here?

Reply:
just below.